recipes-bsp/uefi/tegra-uefi-keys-dtb.bb

```diff
@@ -33,6 +33,14 @@ do_compile() {
     fi
 }
 
+do_populate_sysroot() {
+    install -d ${SYSROOT_DESTDIR}
```

Comment:
Let's follow the normal flow, and make this the `do_install()` task, installing the files under, say, `${D}${datadir}/tegra-uefi-keys`.  Then the check above would look for the file under `os.path.join(d.getVar('STAGING_DATADIR'), 'tegra-uefi-keys')`.